src/main/resources/ColumnDefinition.json

```diff
@@ -34,5 +34,11 @@
   {
     "name": "validationErrors",
     "transferMethod": "LARGETEXT"
+  },
+  {
+    "name": "substudyMemberships",
+    "maximumSize": 250,
+    "transferMethod": "STRING",
```

Comment:
As per the other code review, this will be saved in Dynamo as a Map. Also, you have a custom serialization format. You may need to write your own transfer method.

Reply:
Switched to retrieve a map